content/docs/reference/templates/functions.md

```diff
@@ -1214,6 +1214,7 @@ Creates a complex message object for use in [sendMessage](#sendmessage) function
 - `silent`: whether to suppress push and desktop notifications.
 - `menus`: a single [select menu object](#cmenu).
 - `buttons`: a slice of [button objects](#cbutton).
+- `forward`: must contain an sdict with a channel and message value
```

Comment:
"`forward`: a sdict containing `channel` and `message` keys, specifying the message to forward"?